test/jdk/javax/swing/JFileChooser/FileChooserSymLinkTest.java

```diff
@@ -21,86 +21,124 @@
 
     public static void main(String[] args) throws Exception {
-        Robot robot = new Robot();
         SwingUtilities.invokeAndWait(new Runnable() {
             public void run() {
                 try {
                     initialize();
-                } catch (InterruptedException e) {
-                    throw new RuntimeException(e);
-                } catch (InvocationTargetException e) {
+                } catch (InterruptedException | InvocationTargetException e) {
                     throw new RuntimeException(e);
                 }
             }
         });
-
-        robot.delay(2000);
-        robot.waitForIdle();
-
         passFailJFrame.awaitAndCheck();
     }
 
     static void initialize() throws InterruptedException, InvocationTargetException {
+        //Initialize the components
         final String INSTRUCTIONS = """
                 Instructions to Test:
-                1. Create a regular directory in any specific path.
-                    ex: mkdir C:\\target
-                2. Create a Symbolic link targeting the created test directory.
-                    ex : mklink /D C:\\link C:\\target
-                3. In JFileChooser, navigate to "link" created directed.
-                4. On click of the "link" directory, if the Absolute path of
-                    Symbolic Link is valid then Click PASS, else if it is
-                    null then Click FAIL.(Including MultiSelection Mode SET)
+                1. Open an elevated Command Prompt.
+                2. Paste the following commands:
+                cd /d C:\
+                mkdir FileChooserTest
+                cd FileChooserTest
+                mkdir target
+                mklink /d link FileChooserTest
+
+                3. Navigate to C:\\FileChooserTest in the JFileChooser.
+                4. Use "Enable Multi-Selection" checkbox to enable/disable
+                   MultiSelection Mode
+                5. MultiSelection Mode disabled - On click of the "link"
+                   directory, if the Absolute path of Symbolic Link is
+                   valid then Click PASS, else if it is null then Click FAIL.
+                   MultiSelection Mode Enabled - If Multi selection of
+                   directories including Symbolic Link is possible along
+                   with valid Absolute path then Click PASS, else either
+                   of them failed click FAIL.
+                6. When done with testing, paste the following commands to remove the 'filechooser' directory
+                cd \
+                rmdir /s /q filechooser
+
+                or use File Explorer to clean it up.
                 """;
         frame = new JFrame("JFileChooser Symbolic Link test");
+        panel = new JPanel(new BorderLayout());
+        checkMSelection = new JCheckBox("Enable Multi-Selection");
+        textArea = new JTextArea();
+        jfc = new JFileChooser();
         passFailJFrame = new PassFailJFrame(INSTRUCTIONS);
```

Comment:
```suggestion
        passFailJFrame = new PassFailJFrame("Test Instructions",
                                            INSTRUCTIONS,
                                            5L,
                                            35, 40);
```

Since the instructions are long for this test, I suggest passing additional parameters and modifying the default number of rows displayed so that the complete instructions are displayed.

If you change the instruction text, you may need to amend the number of rows (35).